js-weather/js-challenge1.js

```diff
@@ -0,0 +1,64 @@
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
ソースファイル全体でfunctionキーワードを廃止してアロー関数に置き換えましょう

```suggestion
document.addEventListener('DOMContentLoaded', () => {
```